### ISSUE TITLE
Setup AWS KMS SECP_256k1 private key

### ISSUE THREAD

**Issue by weiyuan95** [WavesHQ/bridge#182]
Flow reference: #177

#### What would you like to be added:
- Setup an SECP_256k1 private key on AWS KMS
- Key should be secure, with proper IAM rules to manage access, with roles for
  - Server
  - Any necessary ad-hoc access for manual intervention, potentially driven driven through scripts and only used locally.

#### Why is this needed:
We need a secure way to store a private key to sign messages for a claim, and to initialise a HD wallet to create multiple addresses for users to deposit funds to.

#### Blocked
This is currently blocked since we do not know how we are going to handle our AWS resources for the bridge.

**Comment by thedoublejay** [comment]
For this, it will only be available on MainNet once BR becomes a part of Consortium. For testing/development purposes, on Playground (Regtest) we can specify a dummy mnemonic first.

@fuxingloh will be reviewing the implementation before go live.

**Comment by weiyuan95** [comment]
> For this, it will only be available on MainNet once BR becomes a part of Consortium. For testing/development purposes, on Playground (Regtest) we can specify a dummy mnemonic first.
> 
> @fuxingloh will be reviewing the implementation before go live.

Got it. We'll still need a place to store the key (i'm guessing AWS KMS) though, unless there is some other way we can sign consortium transactions?

**Comment by weiyuan95** [comment]
For local testing, we can use https://github.com/nsmithuk/local-kms to stub kms behaviour instead of waiting for an actual account that has been provision with KMS rights. This should move dev speed along.

**Comment by weiyuan95** [comment]
@thedoublejay we can probably close this? Since we won't be using KMS to store the key. Unless you want to reword this to AWS secrets manager instead

**Comment by thedoublejay** [comment]
Yep! I will close this for now, we'll recreate when needed
